crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -679,6 +684,45 @@ where
             remote_handle
         };
 
+        tokio::spawn({
```

Comment:
🤔 I guess a task is working for the coordinator, but for the mobile app we may need to trigger it on startup? or allow for manually triggering that sync? I don't think users will have the app open for 30 minutes.

Reply:
This is triggered on start-up inherently 😄

I agree that exposing a different mechanism for app users would also be useful. But for now we can just tell them to restart the app if they want to ensure that missing funds are claimed? I didn't want to get into adding stuff to the frontend with this change.